src/Controller/EnMarche/SummaryController.php

```diff
@@ -26,17 +27,12 @@ class SummaryController extends Controller
     public function indexAction(Summary $summary)
     {
         $this->disableInProduction();
-
-        $pathImage = 'images/'.$summary->getMemberUuid().'.jpg';
-        $signature = SignatureFactory::create($this->getParameter('kernel.secret'))->generateSignature($pathImage, []);
+        $urlPhoto = $this->get(SummaryManager::class)->getUrlProfilePicture($summary, $this->getParameter('kernel.secret'));
 
         return $this->render('summary/index.html.twig', [
             'summary' => $summary,
             'recent_activities' => $this->get(MemberActivityTracker::class)->getRecentActivitiesForAdherent($summary->getMember()),
-            'url_photo' => $this->generateUrl('asset_url', [
-                'path' => $pathImage,
-                's' => $signature,
-            ]),
+            'url_photo' => $urlPhoto,
```

Comment:
The manager should call a setter on the object (the property does not have to be persisted) to remove this logic from controllers, the getter will be called from templates directly.